website/src/docs/hotchocolate/v11/fetching-data/dataloader.md

```diff
@@ -113,13 +113,13 @@ It executes resolvers until the queue is empty and then triggers the data loader
 
 # Data Consistency
 
-Dataloader do not only batch calls to the database, they also cache the database response.
+Data loaders do not only batch calls to the database, they also cache the database response.
```

Comment:
```suggestion
DataLoader do not only batch calls to the database, they also cache the database response.
```

(I still prefer `DataLoaders` [plural], but Michael is likely to insist on `DataLoader` being the plural. 😄)